packages/peregrine/lib/talons/SearchBar/useSuggestedCategory.js

```diff
@@ -0,0 +1,40 @@
+import { useCallback } from 'react';
+
+// TODO: derive from store config when available
```

Comment:
What exactly are we supposed to derive from the config? We're currently querying the config when we first render.

Can you provide specifics in this TODO and/or link to an issue containing specifics :D

Reply:
I don't know; this comment was already there and I preserved it.